test/models/job_test.rb

```diff
@@ -64,6 +64,21 @@ class JobTest < ActiveSupport::TestCase
     job.calculate_retry_delay.must_equal 7.days
   end
 
+  it 'can have retries scheduled' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.